common/src/chain/transaction/input.rs

```diff
@@ -24,7 +25,7 @@ pub enum OutPointSourceId {
     #[codec(index = 0)]
     Transaction(Id<Transaction>),
     #[codec(index = 1)]
-    BlockReward(Id<Block>),
+    BlockReward(Id<GenBlock>),
```

Comment:
Genesis block UTXOs are addressed in a way similar to block rewards rather than by transaction IDs (genesis no longer contains tranctions, just the intial set of unspent outputs)